### PR TITLE
Maven2gradle

### PR DESCRIPTION
**Pull request by sergeyarhutik**

### CODE REVIEW

File: repair-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  
+  repositories {
+    mavenLocal()
+    
+    maven { url "https://repo.maven.apache.org/maven2" }
+  }
+
+  
+  dependencies {
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc', version:'2.2.0.RELEASE'
```
Comment (aleksieiev) [inline repair-service/build.gradle]: version should be collected in one place

File: repair-service/build.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
+allprojects  {
+  apply plugin: 'maven'
+
+  group = 'com.epam'
+version = '0.0.1-SNAPSHOT'
+}
+
+subprojects {
+  apply plugin: 'java'
+  sourceCompatibility = 1.5
```
Comment (aleksieiev) [inline repair-service/build.gradle]: wtf?
Java 1.5?
really?

File: repair-service/build.gradle [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  targetCompatibility = 1.5
+  tasks.withType(JavaCompile) {
+  	options.encoding = 'UTF-8'
+  }
+
+  
+  repositories {
+    mavenLocal()
+    
+    maven { url "https://repo.maven.apache.org/maven2" }
```
Comment (aleksieiev) [inline repair-service/build.gradle]: `mavenCentral()` instead

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile "org.springframework.boot:spring-boot-starter-data-jdbc:*"
```
Comment (aleksieiev) [inline build.gradle]: why `*`?

**Review by aleksieiev — CHANGES_REQUESTED** [review]

**Comment by sergeyarhutik** [comment]
moved all modules to the root dir

**Review by RomanukAlex — APPROVED** [review]

**Pull request closed without merging**